Formula/orocos-kdl.rb

```diff
@@ -20,6 +20,8 @@ class OrocosKdl < Formula
   depends_on "cmake" => :build
   depends_on "eigen"
 
+  patch :DATA
+
```

Comment:
Can we just do this instead?
```suggestion
  patch do
    url "https://github.com/orocos/orocos_kinematics_dynamics/commit/ef39a4fd5cfb1400b2e6e034b1a99b8ad91192cf.patch?full_index=1"
    sha256 "something"
  end

```

Reply:
That works, too! I pushed your suggestion with the actual hash.